packages/autocomplete-plugin-algolia-insights/src/types/AutocompleteInsightsApi.ts

```diff
@@ -1,9 +1,20 @@
 import { createSearchInsightsApi } from '../createSearchInsightsApi';
 
+import type { AlgoliaInsightsHit } from './AlgoliaInsightsHit';
+
 export type AutocompleteInsightsApi = ReturnType<
   typeof createSearchInsightsApi
 >;
 
+export type InsightsParamsWithItems<TParams extends { objectIDs: string[] }> =
+  Omit<TParams, 'objectIDs'> & {
+    items: AlgoliaInsightsHit[];
```

Comment:
I believe this is called `hits` in InstantSearch.js (but a numbered argument, not a named one). I see why items may be more consistent in autocomplete though